EZSwiftExtensionsTests/DoubleTests.swift

```diff
@@ -42,6 +42,15 @@ extension DoubleTests {
         XCTAssertNotEqual(Double(anotherDouble.toInt), anotherDouble)
     }
     
+    func testCGFloat() {
+        XCTAssertEqual(double.toCGFloat, 3.14)
+        XCTAssertEqual(otherDouble.toCGFloat, -147.9564)
+        XCTAssertTrue(double.toCGFloat is CGFloat)
```

Comment:
The main thing that is changed in this function is the type of the variable, so I thought there's a need to explicitly test it.
The Swift compiler is saying that this is not necessary. Should I remove it?